src/test/resources/feature/SignUp.feature

```diff
@@ -0,0 +1,52 @@
+@Irina
+Feature: SignUp
+
+  Background:
+    Given "Home" is displayed
+    And user clicks on "Sign in"
+    And "Login" is displayed
+    And user clicks on "Email address field"
+    And user enters "lalalala@emailll.com"
+    And user clicks on "Create an account"
+    And "CREATE AN ACCOUNT" is displayed
```

Comment:
You can replace this with an single step like **user is on Create an account page** and handle all the logic in StepDef

Reply:
Done